File: CPP/Baju.cpp
#include <bits/stdc++.h>
#include "Aksesoris.cpp"

using namespace std;

class Baju: public Aksesoris{

    private:
        string spesies;
        string ukuran;
        string merk;

    public:
        Baju(){
            spesies = "";
            ukuran = "";
            merk = "";
        }

        Baju(int id, string nama_produk, int harga_produk, int stok_produk, string jenis_aksesoris, string bahan_aksesoris, string warna_aksesoris, string spesies, string ukuran, string merk):Aksesoris(id, nama_produk, harga_produk, stok_produk, jenis_aksesoris, bahan_aksesoris, warna_aksesoris){
            this->spesies = spesies;
            this->ukuran = ukuran;
            this->merk = merk;
        }

        string getSpesies(){ return spesies; }
        string getUkuran(){ return ukuran; }
        string getMerk(){ return merk; }

        void setSpesies(string spesies){ this->spesies = spesies; }
        void setUkuran(string ukuran){ this->ukuran = ukuran; }
        void setMerk(string merk){ this->merk = merk; }

        int spesiesLength(){ return spesies.length(); }
        int ukuranLength(){ return ukuran.length(); }
        int merkLength(){ return merk.length(); }

        ~Baju(){
        }
};

File: PHP/Baju.php
<?php

    class Baju extends Aksesoris{

        private $spesies = "";
        private $ukuran = "";
        private $merk = "";
        
        public function __construct($id, $nama_produk, $harga_produk, $stok_produk, $foto_produk, $jenis, $bahan, $warna, $spesies, $ukuran, $merk){
            parent::__construct($id, $nama_produk, $harga_produk, $stok_produk, $foto_produk, $jenis, $bahan, $warna);
            $this->spesies = $spesies;
            $this->ukuran = $ukuran;
            $this->merk = $merk;
        }

        public function setSpesies($spesies){ $this->spesies = $spesies; }
        public function setUkuran($ukuran){ $this->ukuran = $ukuran; }
        public function setMerk($merk){ $this->merk = $merk; }

        public function getSpesies(){ return $this->spesies; }
        public function getUkuran(){ return $this->ukuran; }
        public function getMerk(){ return $this->merk; }

        public function spesiesLen(){ return strlen($this->spesies); }
        public function ukuranLen(){ return strlen($this->ukuran); }
        public function merkLen(){ return strlen($this->merk); }
    }

?>